packages/config/tests/parse/snapshots/tests.js.md

```diff
@@ -133,6 +133,345 @@ Generated by [AVA](https://avajs.dev).
+            "html": {},␊
+            "images": {},␊
+            "js": {}␊
+          },␊
+          "publish": "packages/config/tests/parse/fixtures/parse_backslash_ambiguous",␊
+          "publishOrigin": "default",␊
+          "services": {}␊
+        },␊
+        "functions": {␊
+          "*": {}␊
+        },␊
+        "headers": [],␊
+        "plugins": [],␊
+        "redirects": []␊
+      },␊
+      "configPath": "packages/config/tests/parse/fixtures/parse_backslash_ambiguous/netlify.toml",␊
+      "context": "production",␊
+      "env": {␊
+        "BRANCH": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "branch"␊
+        },␊
+        "CACHED_COMMIT_REF": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "HEXADECIMAL_ID"␊
+        },␊
+        "COMMIT_REF": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "HEXADECIMAL_ID"␊
+        },␊
+        "CONTEXT": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "production"␊
+        },␊
+        "GATSBY_TELEMETRY_DISABLED": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "1"␊
+        },␊
+        "HEAD": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "branch"␊
+        },␊
+        "LANG": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "en_US.UTF-8"␊
+        },␊
+        "LANGUAGE": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "en_US:en"␊
+        },␊
+        "LC_ALL": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "en_US.UTF-8"␊
+        },␊
+        "NETLIFY_LOCAL": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "true"␊
+        },␊
+        "NEXT_TELEMETRY_DISABLED": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "1"␊
+        },␊
+        "PULL_REQUEST": {␊
+          "sources": [␊
+            "general"␊
+          ],␊
+          "value": "false"␊
+        }␊
+      },␊
+      "headersPath": "packages/config/tests/parse/fixtures/parse_backslash_ambiguous/_headers",␊
+      "redirectsPath": "packages/config/tests/parse/fixtures/parse_backslash_ambiguous/_redirects",␊
+      "repositoryRoot": "packages/config/tests/parse/fixtures/parse_backslash_ambiguous",␊
+      "siteInfo": {}␊
+    }`
+
+## Configuration file - valid backslash in TOML
+
+> Snapshot 1
+
+    `{␊
+      "accounts": [],␊
+      "addons": [],␊
+      "branch": "branch",␊
+      "buildDir": "packages/config/tests/parse/fixtures/parse_backslash_valid",␊
+      "config": {␊
+        "build": {␊
+          "environment": {},␊
+          "ignore": "test /external/path /" /' /external/path /external/path /external/path /external/path /external/path /external/path /external/path /external/path",␊
```

Comment:
Test snapshot normalization is turning those backslash sequences into `/external/path`. :-/